src/test/ui/associated-types/associated-types-for-unimpl-trait.stderr

```diff
@@ -1,8 +1,11 @@
 error[E0277]: the trait bound `Self: Get` is not satisfied
-  --> $DIR/associated-types-for-unimpl-trait.rs:10:8
+  --> $DIR/associated-types-for-unimpl-trait.rs:10:40
    |
+LL | trait Get {
+   | --------- required by this bound in `Get`
```

Comment:
We should (maybe in another PR) customize the label so that it isn't talking about a trait bound when pointing at a trait.